app/controllers/map.ts

```diff
@@ -35,15 +35,6 @@ export default class MapComponent extends Controller {
 
   @tracked path: any = geoPath().projection(this.projection);
 
-  @computed(
```

Comment:
Nooooo idea why these were there. Computeds are old school stuff and never used on methods 😄.